exploring.rst

```diff
@@ -49,20 +50,22 @@ Here's a sample of resources about the architecture of CPython aimed at
 building your understanding of both the 2.x and 3.x versions of CPython:
 
 
-===================================================================================== ================== =========
-               Title                                                                        Author        Version
-===================================================================================== ================== =========
-`Internals of CPython 3.7`_ (using gdb)                                               Louie Lu            3.7.a0
+.. csv-table:: **Current references**
+   :header: "Title", "Brief", "Author", "Version"
+   :widths: 40, 50, 10, 10
 
-`Yet another guided tour of CPython`_                                                 Guido van Rossum    3.5
+    `Internals of CPython 3.7`_ (using gdb), "A guide from AST to Objects, using GDB to observed", Louie Lu, 3.7.a0
```

Comment:
This title needs some restructuring, something like "A guide from AST to Objects, observed using GDB."